docs/optimizations/CheckpointFiles.md

```diff
@@ -4,27 +4,28 @@
 
 Available in Cromwell 55 and higher.
 
-This optimization hopes to resolve the issue of your worker VM being preempted 9 hours and 55 minutes into the runtime of
+This optimization aims to resolve the issue of your worker VM being preempted 9 hours and 55 minutes into the runtime of
 a 10 hour job and having no option but to re-run the entire computation again.
 
 ### Description
 
 Specifying a `checkpointFile` value in a task's `runtime` section designates a checkpoint file which will periodically be
-copied to cloud storage every 10 minutes.
-This checkpoint file will then be restored automatically on subsequent attempts if the job is interrupted.
+copied to cloud storage every 10 minutes. This checkpoint file will then be restored automatically on subsequent attempts if the job is interrupted.
 
-**Note:** Although the checkpoint file is deleted if the task succeeds, additional charges may accrue storing the checkpoint file during
-the running of the task, if the task is aborted or otherwise stopped externally, and by transferring it between the VM and the cloud. These
+To use this feature effectively, the WDL task must be written intentionally to use the checkpoint file. See example below. 
+
+**Note:** Although the checkpoint file is deleted if the task succeeds, storage charges may accrue from (1) storing the checkpoint file during
+the running of the task, (2) aborting or otherwise stopping the task externally, and (3) transferring it between the VM and the cloud storage bucket. These
```

Comment:
Interesting - I think (2) might be worth separating out as a subtly different case.

(1) and (3) are both incidental costs that will happen during normal operation. (2) is saying "and if you abort the task, the checkpoint file will never be cleared up automatically and you'll pay for its storage indefinitely"

Reply:
Oh yes that's a great point, will tweak